src/client/components/Footer/Footer.tsx

```diff
@@ -61,7 +64,11 @@ const Footer: React.FC = () => {
         </a>
       </div>
 
-      <span className="copyright">&copy; FAO, {new Date().getFullYear()}</span>
+      <div className="footer__version">
```

Comment:
Better to show before copyright then | Platform version vX.x.x-duddd-date

Reply:
> Better to show before copyright then | Platform version vX.x.x-duddd-date 

Or maybe even below copyright with smaller font size and lighter grey .

Reply:
Styling updated, added new config in scss:
`$font-xxxs: 10px;` for smaller fonts